blogposts/announcing-sourcegraph-3.10.md

```diff
@@ -0,0 +1,198 @@
+
+Autocomplete suggestions make it easier to use filters in your Sourcegraph searches. Use `ctrl-space` to trigger suggestions for filters you can apply to your query. As you type, the suggestions become more relevant, helping you to apply correct filter options or autocomplete repository or file names.
+
+## Sourcegraph provides native code intelligence to GitLab
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/372226334/de668e24fa?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/372226334/de668e24fa" target="_blank">View on Vimeo</a></p>
+</p>
+
+Sourcegraph users use the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension) to take advantage of go-to-definition and find references on their code hosts. GitLab recognizes the value this brings to developer productivity and is trusting Sourcegraph to provide this functionality natively to their users. Read our [feature announcement](https://about.sourcegraph.com/blog/gitlab-integrates-sourcegraph-code-navigation-and-code-intelligence) and [GitLab’s post](https://about.gitlab.com/blog/2019/11/12/sourcegraph-code-intelligence-integration-for-gitlab/) about collaborating in the open.
+
+Soon, GitLab.com users will be able to opt-in to native code intelligence for public repositories by enabling Sourcegraph in their user preferences. This feature is being rolled out incrementally to public projects on GitLab.com, and will first be available on the [GitLab repository](https://gitlab.com/gitlab-org/gitlab).
+
+Users with private GitLab instances or repositories can enable Sourcegraph code intelligence as of GitLab 12.5. Take a look at the [Sourcegraph integration documentation](https://gitlab.com/gitlab-org/gitlab/blob/master/doc/integration/sourcegraph.md) for how to configure it on your instance.
+
+## Create cross-repository search and replace campaigns
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Sourcegraph now supports creating and tracking code-aware search and replace campaigns on GitHub and Bitbucket Server. When running a search and replace campaign, you preview the changes, then generate the corresponding pull requests on your GitHub and/or Bitbucket Server instances. Once created, you can track the PRs to completion in a burndown chart.
```

Comment:
I think something like this sounds more like "look what you can do!" instead of "here's what you have to do":

```suggestion
Sourcegraph now supports creating and tracking code-aware search and replace campaigns on GitHub and Bitbucket Server. When running a search and replace campaign, you can preview the changes across repositories and then generate the corresponding pull requests on your GitHub and/or Bitbucket Server instances. Once created, you can track the PRs to completion in a burndown chart.
```

Reply:
Outdated version, so updated manually!